pvacseq/lib/main.py

```diff
@@ -52,13 +52,22 @@ def main(args_input = sys.argv[1:]):
 
     args = parser.parse_args(args_input)
 
-    fasta_file = args.sample_name + "_" + str(args.peptide_sequence_length) + ".fa"
-    fasta_key_file = args.sample_name + "_" + str(args.peptide_sequence_length) + ".key"
+    print("Converting VCF to TSV")
+    tsv_file = args.sample_name + '.tsv'
+    lib.convert_vcf.main(
+        [
+            args.input,
+            tsv_file,
```

Comment:
Sorry, I know I already +1'd this, but I checked out the PR to test these changes in conjunction with my PR for the better pvacsec test, and I noticed that the .tsv is placed in the project root directory.  The output tsv filepath should be within the defined output directory